docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+services:
+  app:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    command: >
+      sh -c "python manage.py wait_for_db &&
+             python manage.py migrate &&
+             python manage.py runserver 0.0.0.0:8000"
+    depends_on:
+      - db
+
+  db:
+    image: postgres:16.0-alpine3.17
```

Comment:
Consider adding a `volumes` section for the `db` service to persist database data. This will ensure that data is not lost when the container is stopped or removed.